internal/attestation/renderer/renderer.go

```diff
@@ -55,6 +51,12 @@ func WithLogger(logger zerolog.Logger) Opt {
 	}
 }
 
+func WithSigner(signer sigstoresigner.Signer) Opt {
```

Comment:
we should probably make signer now a required parameter instead of an optional one. On another note, is `keypath` arg used?